sqs_workers/__version__.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (0, 5, 6)
+VERSION = (0, 6, 0)
```

Comment:
Nope, that should be in a separate PR :)

Reply:
👍 Done